### DISCUSSION TITLE
Can't find org.devilutionx folder in Android/data

### DISCUSSION THREAD

**Discussion by ablumenberg** [diasurgical/DevilutionX#7000] [Q&A]
### Operating System

Android

### DevilutionX version

1.5.2

### Describe

This is a great app! The demo of diablo runs really well and I'm excited to play the full game. 

 When I open the file system on my PC and go to Android/data/ the org.devilutionx folder is not present. I tried uninstalling and reinstalling the app, and looking in other folders on my internal storage. Is this a known issue or just my device?
thanks!

### To Reproduce

Installed the devilutionx app on my android phone
connected phone to my PC and navigated to Internal storage\Android\data
Folders for my other applications are visible, there is no folder for diasurgical or devilutionX

### Expected Behavior

I expected there to be a corresponding folder wherein I could copy my Diablo 1 game file. 

### Additional context

Impressive app! Thanks!!

**Comment by DakkJaniels** [comment]
duplicate of #6975 ?

**Comment by AJenbo** [comment]
This is not the right place for support questions. I have converted it to a discussion instead.

It sounds like the device you are using (which one) has more than one storage device and the app is probably using the other one than the one you checked.

**Comment by StephenCWills** [comment]
The folder is called `org.diasurgical.devilutionx`, and it may not be created until after the first time you run the app.

**Comment by ablumenberg** [comment]
I figured out what I was doing incorrectly. The correct thing to do is:

1) Transfer the DIABDAT.MPQ file to any accessible folder in the android system. I used the downloads folder.
2) Long press on the DevilutionX icon and select the file.

The developers made it a simple process.
